bundle.Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG quay_expiration=never
+ARG release_tag=0.0.0
```

Comment:
that's the sneaky trick :)
make bundle on existing manifests will update the version dynamically according to what we pass here.
while the manager image will have the `VERSION` specified in makefile.

Reply:
Btw it will also add api changes to the csv.
saving an untarring a recent local build - I found out it added your newest attributes from crd to the csv.
I haven't run `make bundle` after git syncing the code.   the build did - and it picked up the changes.